metric/metric.go

```diff
@@ -22,6 +26,21 @@ type InstanceCountMetric struct {
 
 // IssueCountMetric is the API message for bb.issue.count
 type IssueCountMetric struct {
-	Type  api.IssueType
-	Count int
+	Type   api.IssueType
+	Status api.IssueStatus
+	Count  int
+}
+
+// ProjectCountMetric is the API message for project count metric
+type ProjectCountMetric struct {
+	TenantMode   api.ProjectTenantMode
+	WorkflowType api.ProjectWorkflowType
+	Count        int
+}
+
+// PolicyCountMetric is the API message for policy count metric
+type PolicyCountMetric struct {
+	Type          api.PolicyType
```

Comment:
Mere policy is only interesting for schema review policy (since empty means no schema review policy is enforced for that environment), so calculating the policy count doesn't seem to be useful.

For backup and approval policy, the interesting part is the policy value.

So for the policy metrics, we need more granular metrics. The policy type is also the policy metric name.

Reply:
We can also collect policy metrics later. The more important item is to collect task-related metrics

Reply:
I see. As one env is forced to have one `backup-plan` policy and `pipeline-approval`, what we need is the policy setting instead of the policy type.